src/main/java/socchat/TaskList.java

```diff
@@ -0,0 +1,139 @@
+package socchat;
+
+import socchat.task.Task;
+import socchat.task.deadline.Deadline;
+import socchat.task.event.Event;
+import socchat.task.todo.Todo;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+
+public class TaskList {
+    private Parser parser = new Parser();
+    private Storage storage;
+    private ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void delete(String indexString) throws SocchatException {
+        try {
+//            String taskIndexString = scanner.nextLine().trim();
```

Comment:
Perhaps the comment here should be indented relative to their position in the code as per the coding standards? 

```
while (true) {
    // Do something
    something();
}
```